docs/pages/server-access/guides/openssh.mdx

```diff
@@ -395,3 +395,66 @@ $ tsh ssh --port=22 dev@database.root.mytenant.teleport.sh
 To revoke the current Teleport CA and generate a new one, run `tctl auth rotate`. Unless you've highly automated your
 infrastructure, we would suggest you proceed with caution as this will invalidate the user
 and host CAs, meaning that the new CAs will need to be exported to every OpenSSH-based machine again using `tctl auth export` as above.
+
+## Connect to leaf clusters using jump proxies
+
+Normally, you connect to leaf cluster SSH nodes through the root cluster proxy.
+In scenarios where leaf proxies are reachable by SSH clients, you might prefer
+to connect directly through the leaf proxies to avoid potential latency issues
+of the connection being routed through the root cluster.
```

Comment:
nit: It might be worth mentioning that the "potential latency issues" are only relevant in regionally distributed clusters, if I'm understanding correctly.
```suggestion
However, in regionally distributed clusters where leaf proxies are reachable by
SSH clients, you might prefer to connect directly through the leaf proxies to avoid
potential latency issues of the connection being routed through the root cluster.
```